invisible_cities/reco/hits_functions_test.py

```diff
@@ -108,26 +113,34 @@ def test_threshold_hits_with_penthesilea(config_tmpdir, Kr_pmaps_run4628_filenam
     hits_pent_th1 = hio.load_hits(PATH_OUT_th1)
     hits_pent_th2 = hio.load_hits(PATH_OUT_th2)
     ev_num = 1
-    hits_thresh  = threshold_hits (hits_pent_th1[ev_num].hits, th=th2)
+    hits_thresh  = threshold_hits (hits_pent_th1[ev_num].hits, th = th2)
     assert len(hits_pent_th2[ev_num].hits)==len(hits_thresh)
-    for h1, h2 in zip(hits_pent_th2[ev_num].hits,hits_thresh):
-        assert_almost_equal (h1.E, h2.E, 3)
-        assert_almost_equal (h1.Q, h2.Q, 3)
-        assert h1.Z==h2.Z
-        assert h1.X==h2.X
-        assert h1.Y==h2.Y
+    for h1, h2 in zip(hits_pent_th2[ev_num].hits, hits_thresh):
+        assert_hit_equality(h1, h2)
+
 
 @given(list_of_hits(), floats())
 def test_threshold_hits_not_modify_input(hits, th):
-    hits_org=deepcopy(hits)
-    before_len = len(hits)
+    hits_org    = deepcopy(hits)
+    before_len  = len(hits)
     hits_thresh = threshold_hits(hits,th)
-    after_len = len(hits)
+    after_len   = len(hits)
     assert before_len == after_len
     for h1, h2 in zip(hits, hits_org):
         assert_hit_equality(h1, h2)
 
+
 @given(list_of_hits(), floats())
-def test_threshold_hits_energy_conserved(hits,th):
-    hits_thresh = threshold_hits(hits,th)
-    assert_almost_equal(sum((h.E for h in hits)), sum((h.E for h in hits_thresh)))
+def test_threshold_hits_energy_conserved(hits, th):
+    hits_thresh   = threshold_hits(hits, th)
+    assert_almost_equal(sum((h.E for h in hits)), sum((h.E for h in hits_thresh  )))
+    hits_thresh_c = threshold_hits(hits, th)
```

Comment:
Are these two variables `hits_thresh` and `hits_thresh_c` intended to be equal? It looks to me that they are, since they are both the result of `threshold_hits(hits, th)` with the `uncorrected` argument being False. I'm probably missing the meaning of this test.

Reply:
Because it indeed has no meaning, I corrected hits_thresh_c to be on corrected hits